score-lang-cli/src/main/java/org/openscore/lang/cli/services/ScoreServicesImpl.java

```diff
@@ -124,19 +124,51 @@ public synchronized void onEvent(ScoreEvent scoreEvent) throws InterruptedExcept
                     break;
                 case ScoreLangConstants.EVENT_INPUT_END:
                     String taskName = (String)data.get(LanguageEventData.levelName.TASK_NAME.name());
+
                     if(StringUtils.isNotEmpty(taskName)){
-                        String path = (String) data.get(LanguageEventData.PATH);
-                        int matches = StringUtils.countMatches(path, ExecutionPath.PATH_SEPARATOR);
-                        String prefix = StringUtils.repeat(TASK_PATH_PREFIX, matches);
+                        String prefix = getPrefix(data);
                         printWithColor(Ansi.Color.YELLOW, prefix + taskName);
                     }
                     break;
+                case ScoreLangConstants.EVENT_OUTPUT_END:
+                    printOutputs(data);
+                    break;
+
                 case EVENT_EXECUTION_FINISHED :
                     printFinishEvent(data);
                     break;
             }
         }
 
+        private void printOutputs(Map<String, Serializable> data) {
+            if(data.containsKey(LanguageEventData.OUTPUTS) && data.containsKey(LanguageEventData.PATH) && data.get(LanguageEventData.PATH).equals("0/0")) {
+
+                @SuppressWarnings("unchecked") Map<String, String> outputs = (Map<String, String>) data.get(LanguageEventData.OUTPUTS);
+
+                String prefix = getPrefix(data);
+
+                if (outputs != null && !outputs.keySet().isEmpty()) {
+
+                    printWithColor(Ansi.Color.WHITE, prefix + "Outputs:");
+
+                    for (String key : outputs.keySet()) {
+                        String outputValue = outputs.get(key).replace("\n", " ");
+                        outputs.put(key, StringUtils.abbreviate(outputValue, 0, OUTPUT_VALUE_LIMIT));
+                        if (StringUtils.isEmpty(outputValue)) {
+                            outputs.put(key, "(empty)");
+                        }
+                        printWithColor(Ansi.Color.WHITE, prefix + "  " + key + " = " + outputs.get(key));
+                    }
```

Comment:
no we don't need it anymore. Though I suggest printing like we were to get the prefix so it looks better.